shared/reducers/profile.tsx

```diff
@@ -146,4 +146,10 @@ export default Container.makeReducer<ProfileGen.Actions, Types.State>(initialSta
   [ProfileGen.submitZcashAddress]: draftState => {
     updateUsername(draftState)
   },
+  [ProfileGen.wotVouch]: draftState => {
+    draftState.wotAuthorError = ''
+  },
```

Comment:
Just seeing this now.

In `actions/profile/index.tsx`, would you want to dispatch `ProfileGen.createWotVouch` at the end (on success) rather than `ProfileGen.createWotVouchError({ error: '' })`?

Reply:
`wotVouch` means: The user has clicked Submit. Saga, fire the RPC. Which will cause the waiting key.

On RPC success: Saga dispatches wotVouchError("") and closeModals

On RPC error: Saga dispatches wotVouchError(rpc.error)

wotVouchError only sets the store value

So without further changes dispatch wotVouch on success doesn't make sense since success only happens as a result of wotVouch's RPC.